lib/index.cjs

```diff
@@ -3,21 +3,21 @@
 const checkAgainstRule = require('./utils/checkAgainstRule.cjs');
 const createPlugin = require('./createPlugin.cjs');
 const createStylelint = require('./createStylelint.cjs');
-const formatters = require('./formatters/index.cjs');
+const index$1 = require('./formatters/index.cjs');
```

Comment:
I searched for an option you thought, but unfortunately, I cannot find it. (Please let me know if you find it instead 🙏🏼 )
https://rollupjs.org/configuration-options/

The simplest way could be to rename from `formatters/index.mjs` to `formatters.mjs`, but I'd like to avoid doing it for now so that we could easily trace commit history.